plugins/dell/fu-dell-smi.c

```diff
@@ -27,7 +27,7 @@ static void
 _dell_smi_obj_free (FuDellSmiObj *obj)
 {
 	dell_smi_obj_free (obj->smi);
-	g_free(obj);
+	g_free (obj);
```

Comment:
Lets drop the reformatting for this PR. We can apply manually it on new plugins or when we refactor files.

Reply:
Done